udev/readme.md

```diff
@@ -0,0 +1,10 @@
+
+# Installing udev
+
+## Ubuntu 22.04 (likely other debian variants)
```

Comment:
It works exactly the same for me on Fedora - so I think it's safe to say it's the same on any (modern) distro.